<!-- ReasonProcedureNotAttemptedRelationship -->

<pattern id="p-PatientCareReport_1A_ReasonProcedureNotAttemptedRelationship_ReasonProcedureNotAttemptedObservation_4_105_15_i-error_mdht" see="#p-PatientCareReport_1A_ReasonProcedureNotAttemptedRelationship_ReasonProcedureNotAttemptedObservation_4_105_15_i-error_mdht" name="p-PatientCareReport_1A_ReasonProcedureNotAttemptedRelationship_ReasonProcedureNotAttemptedObservation_4_105_15_i-error_mdht">
		<!--
			UML path: /Procedure/ReasonProcedureNotAttemptedRelationship
			UML root path: /PatientCareReport/ProceduresPerformedSection/procedureActivityProcedure1/ReasonProcedureNotAttemptedRelationship
			UML qualified name: emspcr::Procedure::ReasonProcedureNotAttemptedRelationship
		-->
	<rule context="/cda:ClinicalDocument/cda:ProceduresPerformedSection[cda:templateId/@root = '2.16.840.1.1133883.17.3.10.1.21']/cda:procedureActivityProcedure1[cda:templateId/@root = '2.16.840.1.1133883.17.3.10.1.181']/cda:entryRelationship[cda:observation/cda:templateId/@root = '2.16.840.1.1133883.17.3.10.1.137']">
		<!--
			OCL: self.observation->one(observation : cda::Observation | not observation.oclIsUndefined() and observation.oclIsKindOf(emspcr::ReasonProcedureNotAttemptedObservation))
			Constraint name: ProcedureReasonProcedureNotAttemptedRelationshipReasonProcedureNotAttemptedObservation
		-->
	<assert test="count(cda:observation[cda:templateId/@root = '2.16.840.1.1133883.17.3.10.1.137'])=1">Error: PatientCareReport - 4.105.15.i ReasonProcedureNotAttemptedObservation
		Conformance: SHALL contain exactly one [1..1] observation (CONF:11088)
		Analysis: n/a
		Validation message: n/a	</assert>
	</rule>
</pattern>

